src/main/java/com/atlauncher/gui/dialogs/InstanceInstallerDialog.java

```diff
@@ -33,9 +33,7 @@
 import java.awt.event.WindowEvent;
 import java.io.File;
 import java.nio.file.Files;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
+import java.util.*;
```

Comment:
Would you be able to undo this import change. I prefer to always have explicit imports for readability